### PR TITLE
Feature/erssup 61504

### PR DESCRIPTION
**Pull request by nhsd-david-wass**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: ecs-proxies-deploy.yml [hunk 1]
```diff
@@ -3,4 +3,4 @@ docker_service:
     expose: true
```
Comment (nhsd-david-wass) [inline ecs-proxies-deploy.yml:3]: This was causing the sandbox to constantly be restarted and is probably the reason we have been having intermittent problems with responses from the sandbox for some time

File: sandbox/src/routes/generatePatientLetter.js [hunk 2]
```diff
@@ -24,7 +24,8 @@ module.exports = [
         if (responsePath && responseCode) {
           return h.file(responsePath, {
             mode: 'attachment',
-            filename: filename
+            filename: filename,
+            etagMethod: false
           }).code(responseCode);
         }
```
Comment (nhsd-david-wass) [inline sandbox/src/routes/generatePatientLetter.js:30]: Added 'etagMethod: false' to stop erroneous etag headers in response

File: sandbox/src/mocks/uploadFileToDocumentStore/requests/upload.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+A simple test upload file
```
Comment (nhsd-david-wass) [inline sandbox/src/mocks/uploadFileToDocumentStore/requests/upload.txt:1]: Add comment in this file to say that it is test only

File: tests/sandbox/test_a029_available_actions_fo_user.py [hunk 4]
```diff
@@ -0,0 +1,134 @@
+from typing import Callable, Dict, Iterable, List
```
Comment (nhsd-jack-wainwright) [inline tests/sandbox/test_a029_available_actions_fo_user.py:1]: Just noticed a minor typo in the name of this file, I think it should be `test_a029_available_actions_for_user.py`

**Review by nhsd-jack-wainwright — COMMENTED** [review]
Looks good to me 🙂 , just a couple very minor comments.

**Review by nhsd-jack-wainwright — APPROVED** [review]

**Pull request merged**